addons/point_of_sale/static/src/app/screens/product_screen/product_screen.xml

```diff
@@ -6,7 +6,7 @@
                 class="leftpane d-flex flex-column border-end bg-200" >
                 <OrderSummary />
                 <div class="pads border-top">
-                    <ControlButtons t-if="!ui.isSmall" />
+                    <ControlButtons t-if="!ui.isSmall" partner="currentOrder?.get_partner()" onClickMore.bind="displayAllControlPopup"/>
```

Comment:
I'm not against passing props but order partner isn't accessible directly from `ControlButtons` ?